test/fuzzing/meson.build

```diff
@@ -34,45 +34,42 @@ foreach file_name : tests
   set_variable('@0@_exe'.format(test_name.underscorify()), exe)
 endforeach
 
-env = environment()
-env.set('srcdir', meson.current_source_dir())
-
-test('shape-fuzzer', find_program('run-shape-fuzzer-tests.py'),
+test('shape-fuzzer', find_program('run-fuzzer-tests.py'),
   args: [
     hb_shape_fuzzer_exe,
+    meson.current_source_dir() / 'fonts',
   ],
   depends: [hb_shape_fuzzer_exe, libharfbuzz, libharfbuzz_subset],
   workdir: meson.current_build_dir() / '..' / '..',
-  env: env,
   priority: 1,
   suite: ['fuzzing'],
 )
 
-test('subset-fuzzer', find_program('run-subset-fuzzer-tests.py'),
+test('subset-fuzzer', find_program('run-fuzzer-tests.py'),
   args: [
     hb_subset_fuzzer_exe,
+    meson.current_source_dir() / 'fonts',
   ],
   workdir: meson.current_build_dir() / '..' / '..',
-  env: env,
   priority: 1,
   suite: ['fuzzing'],
 )
 
-test('repacker-fuzzer', find_program('run-repacker-fuzzer-tests.py'),
+test('repacker-fuzzer', find_program('run-fuzzer-tests.py'),
   args: [
     hb_repacker_fuzzer_exe,
+    meson.current_source_dir() / 'fonts',
```

Comment:
This one should use files in `graph`, no?

Reply:
@garretrieger Are those fonts? Or some custom graph encoding?

Reply:
Right, I missed that. The subset fuzzer uses to dirs as well, let me fix that.